jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletApiRequest.java

```diff
@@ -257,17 +257,7 @@ public AuthenticationState getAuthentication()
 
     private AuthenticationState getUndeferredAuthentication()
```

Comment:
Perhaps this also should be renamed to `getUndeferredAuthenticationState()`?

Are these other cases that miss the `State` suffix?

Reply:
I think I should leave this one as is so it matches the public API directly above.